src/test/java/org/greenbuttonalliance/gbaresourceserver/usage/repository/TimeConfigurationTest.java

```diff
@@ -0,0 +1,115 @@
+package org.greenbuttonalliance.gbaresourceserver.usage.repository;
+
+import com.github.f4b6a3.uuid.UuidCreator;
+import lombok.RequiredArgsConstructor;
+import org.greenbuttonalliance.gbaresourceserver.usage.model.TimeConfiguration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@DataJpaTest(showSql = false)
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class TimeConfigurationTest {
```

Comment:
Both teams need to agree on a consistent Test naming configuration so Test class names use the same format.  For example, all integration tests should end in "IT" and all unit tests should end in "Test".  The rest of the class name should indicate what is being tested.  In this case, since we are testing the TimeConfigurationRepository I agree the class name should be TimeConfigurationRepositoryTest.